thoth/storages/data/alembic/versions/50081eea5da2_adjust_cve_model_to_consume_.py

```diff
@@ -0,0 +1,38 @@
+"""Adjust CVE model to consume vulnerabilities from pypa/advisory-db
+
+Revision ID: 50081eea5da2
+Revises: db6290ac490d
+Create Date: 2021-06-02 08:15:43.474297+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '50081eea5da2'
+down_revision = 'db6290ac490d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("DELETE FROM CVE")
+    op.add_column('cve', sa.Column('details', sa.Text(), nullable=False))
+    op.drop_column('cve', 'version_range')
+    op.drop_column('cve', 'source_url')
```

Comment:
`source_url` is not part of CVE table, the upgrade fails, removing this line and the one below related to `source_url` seems to solve the issues and allow for the migration.